src/App.js

```diff
@@ -58,19 +70,18 @@ class App extends Component {
           <h2 className="page-name">Quizzelydoo</h2>
         </div>
         {isQuizIntro ? (
-          <div className="Introduction">
-            <Adbox />
+          <Transition in={this.state.in} timeout={400} className="exiting">
             <Introquiz
+              className="Quiz-Introduction"
               quiztitle={quizInfo.quizheadline.quiztitle}
               intropic={quizInfo.quizheadline.intropic}
               quizsummary={quizInfo.quizheadline.quizsummary}
               onClick={this.handleQuizStart}/>
-            <Adboxtwo />
-          </div>
+          </Transition>
         ) : (
-          <div className="Quiz-Display">
+          <Transition in={this.onEnter} timeout={400} className="entering">
```

Comment:
Wrapped everything in a div, but also should I just change the className to "questions" or "questions-entering"? Was a little confused about that or how you'd get questions-entered and all that?